main.go

```diff
@@ -174,13 +174,18 @@ func main() {
 		os.Exit(1)
 	}
 }
+
 // WatchNamespaceEnvVar is the constant for env variable WATCH_NAMESPACE
 // which specifies the Namespace to watch.
 // An empty value means the operator is running with cluster scope.
 const watchNamespaceEnvVar = "WATCH_NAMESPACE"
 
 // getWatchNamespace returns the Namespace the operator should be watching for changes
 func getWatchNamespace() (string, bool) {
+	// WatchNamespaceEnvVar is the constant for env variable WATCH_NAMESPACE
+	// which specifies the Namespace to watch.
+	// An empty value means the operator is running with cluster scope.
+	const watchNamespaceEnvVar = "WATCH_NAMESPACE"
```

Comment:
@asoorm  This is only used within the function call, used only once, and can't be changed without looking at the function call (Meaning not meant to easy refactoring). The doc comment looks like a global exported symbol to me.

In this case `os.LookupEnv( "WATCH_NAMESPACE")`  should be fine

Did I miss something?